### ISSUE TITLE
Project layout has some mild issues

### ISSUE THREAD

**Issue by mrdaemon** [ccfreak2k/BESM-Character-Resolver#4]
The way the files are laid out isn't particularily standard. I will (at a previously agreed time as not to break everything) refactor it in the forthcoming weeks.

This will help for things such as:
- Documentation Generation (docutils/sphinx)
- Content and Ressources generation (pyrcc/pyuic)
- Distribute packaging (setup.py)
- Standalone binary 'compilation'.

**Comment by ccfreak2k** [comment]
Having a standard layout made before too many files are added will reduce later work, but part of the problem is that all of the features haven't quite been laid out yet.

**Comment by Knifa** [comment]
Yeah, I don't really know what I'm doing in terms of package layout and so on, or naming conventions for packages.

**Comment by mrdaemon** [comment]
Well either way there pretty much exists a standard for python packages :) I suggest we follow it. I will work on the layout, probably along the setup/build file.
